src/actions/plugins/easymotion/easymotion.cmd.ts

```diff
@@ -79,6 +79,17 @@ abstract class BaseEasyMotionCommand extends BaseCommand {
       // Search all occurences of the character pressed
       const matches = this.getMatches(position, vimState);
 
+      // If previous mode was visual, restore visual selection
+      if (
```

Comment:
Can you use `isVisualMode` here? (Mode.ts)

Reply:
Yes already changed it.